packages/shared/providers.yaml

```diff
@@ -4582,6 +4582,21 @@ personio:
             title: App ID
             description: The app ID of your Personio account
 
+personio-v2:
+    display_name: Personio (v2)
+    categories:
+        - hr
+    auth_mode: OAUTH2_CC
+    proxy:
+        base_url: https://api.personio.de/v2
+        headers:
+            Content-Type: application/x-www-form-urlencoded
+    token_url: https://api.personio.de/v2/auth/token
```

Comment:
@NangoHQ/engineers 
Should make the `version` a `connectionConfig` instead with a default value? so we don't need a v3, v4 etc.

Reply:
I think that would be a bit troublesome in the event they want to upgrade to a newer version of the API. I think we should at some point deprecate al the providers that have a version in the `base_url` and make all customers specify a version in their integration scripts. 

The base url IMO for this should ideally just be

```
base_url: https://api.personio.de
```

Reply:
the problem is the token_url, we need the version there

Reply:
Team agreed that it's too challenging to solve right now